carla-carma-integration/carla_carma_bridge/src/carla_carma_bridge/carla_to_autoware_vehicle_status

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2021 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+"""
+Subscribe from CARLA :carla_msgs::CarlaEgoVehicleStatus
+    Topic: /carla/{}/vehicle_status
+
+Subscribe from CARLA :carla_msgs::CarlaEgoVehicleInfo
+    Topic: /carla/{}/vehicle_info
+
+Publish to CARMA :autoware_msgs::VehicleStatus
+    Topic: /hardware_interface/vehicle_status
+
+"""
+import math
+import rospy
+from carla_msgs.msg import CarlaEgoVehicleInfo
+from carla_msgs.msg import CarlaEgoVehicleStatus
+from autoware_msgs.msg import VehicleStatus, Gear
+
+pub = rospy.Publisher('/hardware_interface/vehicle_status', VehicleStatus, queue_size=1)
+vehicle_info = None
+
+
+def vehicle_info_callback(vehicle_info_msg):
+    """
+    callback for vehicle info
+    vehicle_info_msg type:
+        carla_msgs::CarlaEgoVehicleInfo
+    """
+    global vehicle_info
+    vehicle_info = vehicle_info_msg
+
+def vehicle_status_callback(vehicle_status_msg):
+    """
+    callback for vehicle status
+    vehicle_status_msg type:
+        carla_msgs::CarlaEgoVehicleStatus
+    """
+    if vehicle_info is None:
+        return
+    status = VehicleStatus()
+    status.header = vehicle_status_msg.header
+
+    # calculate max steering angle
+    max_steering_angle = math.radians(70)
```

Comment:
This might be best left as a config value, different vehicles may have different max steering angles.

Reply:
I've updated the script for leaving max_steering_angle parameter as a config value